tracpro/polls/tasks.py

```diff
@@ -92,7 +92,9 @@ def fetch_org_updated_runs(org):
     incomplete_responses = Response.get_update_required(org)
 
     if incomplete_responses:
-        runs = client.get_runs(ids=[r.flow_run_id for r in incomplete_responses])
+        runs = []
+        for i in range(0, len(incomplete_responses), 15):
```

Comment:
Agree with @xkmato's that this task doesn't run frequently enough to not fetch all incomplete runs.

Where did the 15 number come from? I'm wondering if we can't grab something like 50 at a time safely and so keep the number of API calls down. URL length limit is 2048 right?

And as @nicpottier points out - this task needs a lock so if it's still running when it's invoked again, it should do nothing.